sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.students.controller.getAssessmentAwardDetails.e2e-spec.ts

```diff
@@ -163,6 +167,8 @@ describe("AssessmentStudentsController(e2e)-getAssessmentAwardDetails", () => {
           disbursement1sbsd: 666,
         },
         finalAward: {
+          disbursementReceipt1Received: false,
+          disbursementReceipt1HasAwards: true,
```

Comment:
This outcome seems to be weird where the receipt is not received but the receipt has awards. Please let me know if I am missing something here.

Reply:
`disbursementReceipt` is the prefix used for part-time and full-time in the API return. The difference is that full-time really gets the values from the receipt, while part-time "emulates" the receipt values from the e-Cert-generated awards.